library_schemas/slam/prerelease/hedtsv/HED_slam_1.0.0/HED_slam_1.0.0_Prefixes.tsv

```diff
@@ -8,4 +8,5 @@ prov:	http://www.w3.org/ns/prov#	The PROV namespace [PROV-DM]
 dc:	http://purl.org/dc/elements/1.1/	The Dublin Core elements
 terms:	http://purl.org/dc/terms/	The Dublin Core terms
 foaf:	http://xmlns.com/foaf/0.1/	Friend-of-a-Friend http://xmlns.com/foaf/spec/
-has_dbxref	http://www.geneontology.org/formats/oboInOwl#dbxref	Relates an entity to an external database entry
\ No newline at end of file
+obo:	http://purl.obolibrary.org/obo/	Open Biomedical Ontologies
+has_dbxref:	http://www.geneontology.org/formats/oboInOwl#dbxref	Relates an entity to an external database entry
```

Comment:
The has_debxref should not be in the Prefixes file  it should be in the AnnotationsExternal file -- it is not a namespace, it is an AnnotationProperty.  Thus Line 12 should not be there -- However the obo ref should maybe be http://www.geneontology.org/formats/oboInOwl#.  Again, we will need to think about what property we want to use in general for this -- the `has_dbxref`  has a specific meaning.